setup.py

```diff
@@ -57,6 +58,12 @@ def get_new_url(url):
 with open("rig/version.py", "r") as f:
     exec(f.read())
 
+# The requirements vary depending on the version of Python
+requirements = ["numpy>1.6", "six", "sentinel", "pytz"]
+if sys.version_info[0] < 3 or sys.version_info[1] < 4:
```

Comment:
I didn't realise one could do that...

(Not that I'm entirely certain I know how that gets parsed)